tests/rt_35d.conf

```diff
@@ -11,3 +11,12 @@ RUN     | cpld_bmark_p7_35d
 RUN     | cpld_bmark_p7_35d                                                                                               |                             | fv3 |  | 2013040100
 RUN     | cpld_bmark_p7_35d                                                                                               |                             | fv3 |  | 2013070100
 RUN     | cpld_bmark_p7_35d                                                                                               |                             | fv3 |  | 2013100100
```

Comment:
Do we want to turn off all the bmark_p7_35d tests?

Reply:
Yes, I will remove the p7_35d tests and the edit the rt_35d to be only p8.

I will also remove the opnReq logs for p7.

Reply:
@DeniseWorthen @junwang-noaa The cpld_bmark_p7_35d tests are used by the weekly tests.

Reply:
@binli2337 I've updated the rt_weekly and set the fhmax to 5days. I set the wall clock to 120. I think that should be enough. Please check the changes, thanks

Reply:
@DeniseWorthen The updates are good. Thanks.